openvdb/Grid.h

```diff
@@ -1447,7 +1447,16 @@ template<typename TreeT>
 inline void
 Grid<TreeT>::pruneGrid(float tolerance)
 {
-    this->tree().prune(ValueType(zeroVal<ValueType>() + tolerance));
+    // Suppress conversion warning that GCC 6.3+ emits when ValueType is int or long int
+#if defined(__GNUC__)
+  #pragma GCC diagnostic push
+  #pragma GCC diagnostic ignored "-Wconversion"
+#endif
```

Comment:
I'd prefer an `OPENVDB_NO_TYPE_CONVERSION_WARNING_BEGIN`/`OPENVDB_NO_TYPE_CONVERSION_WARNING_END` macro pair (see, e.g., `OPENVDB_NO_UNREACHABLE_CODE_WARNING_BEGIN` in `Platform.h`).  Although these changes are temporary, it might turn out that in some cases suppressing the warnings is the only practical solution, though that should always be a last resort.